specs/src/consensus.md

```diff
@@ -32,6 +32,7 @@
 | `SHARE_INFO_BYTES`                      | `uint64` | `1`          | `byte`  | The number of bytes used for [share](data_structures.md#share) information                                                                                               |
 | `SHARE_RESERVED_BYTES`                  | `uint64` | `4`          | `byte`  | The number of bytes used to store the index of the first transaction in a transaction share. Must be able to represent any integer up to and including `SHARE_SIZE - 1`. |
 | `SHARE_SIZE`                            | `uint64` | `512`        | `byte`  | Size of transaction and blob [shares](data_structures.md#share), in bytes.                                                                                               |
+| `SignerSize`                            | `int`    | `20`         | `byte`  | The number of bytes used to store the signer in a [share](data_structures.md#share).                                                                                     |
```

Comment:
Is it problematic that there's a different formatting i.e. SignerSize and not SIGNER_SIZE?

Reply:
Agreed it's inconsistent but the specs already have inconsistent casing (e.g. `v1.Version`). This PR uses the same casing as the constant defined in Go code. 

Added https://github.com/celestiaorg/celestia-app/issues/3660#issuecomment-2405492855